src/main/java/me/partlysanestudios/partlysaneskies/system/OneConfigScreen.java

```diff
@@ -37,6 +37,10 @@ public void resetBrokenStrings() {
             secretsChatMessageString = "Partly Sane Skies > All required secrets have been found!";
             save();
         }
+        if (pickaxeAbilityReadyBannerText.isEmpty()){
+            pickaxeAbilityReadyBannerText = "Pickaxe Ability Ready!";
```

Comment:
Does this code allow you to modify it at all

Reply:
> Does this code allow you to modify it at all

it should since the other .empty() work the same

Reply:
> > > Does this code allow you to modify it at all
> > 
> > 
> > it should since the other .empty() work the same
> 
> Double check since I think this might have been broken for a while

it works as long as you dont clear out the entire input

Reply:
> > > > Does this code allow you to modify it at all
> > > 
> > > 
> > > it should since the other .empty() work the same
> > 
> > Double check since I think this might have been broken for a while
> 
> it works as long as you dont clear out the entire input

Got it okay

Reply:
> > > > > Does this code allow you to modify it at all
> > > > 
> > > > 
> > > > it should since the other .empty() work the same
> > > 
> > > Double check since I think this might have been broken for a while
> > 
> > it works as long as you dont clear out the entire input
> 
> Got it okay

Quote replies in github are funny

Reply:
> > > > > > Does this code allow you to modify it at all
> > > > > 
> > > > > 
> > > > > it should since the other .empty() work the same
> > > > 
> > > > 
> > > > Double check since I think this might have been broken for a while
> > > 
> > > 
> > > it works as long as you dont clear out the entire input
> > 
> > 
> > Got it okay
> 
> Quote replies in github are funny

im enjoying this too much

Reply:
> > > > > > > > > Does this code allow you to modify it at all
> > > > > > > > 
> > > > > > > > 
> > > > > > > > it should since the other .empty() work the same
> > > > > > > 
> > > > > > > 
> > > > > > > Double check since I think this might have been broken for a while
> > > > > > 
> > > > > > 
> > > > > > it works as long as you dont clear out the entire input
> > > > > 
> > > > > 
> > > > > Got it okay
> > > > 
> > > > 
> > > > Quote replies in github are funny
> > > 
> > > 
> > > im enjoying this too much
> > 
> > Yeah you can leave it like that
> 
> Okay!

Back in my day this is how we had to do it on discord

Reply:
> > > > > > > > > > Does this code allow you to modify it at all
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > it should since the other .empty() work the same
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Double check since I think this might have been broken for a while
> > > > > > > 
> > > > > > > 
> > > > > > > it works as long as you dont clear out the entire input
> > > > > > 
> > > > > > 
> > > > > > Got it okay
> > > > > 
> > > > > 
> > > > > Quote replies in github are funny
> > > > 
> > > > 
> > > > im enjoying this too much
> > > 
> > > 
> > > Yeah you can leave it like that
> > 
> > 
> > Okay!
> 
> Back in my day this is how we had to do it on discord

it wasnt as funny tho since > dont stack there

Reply:
> > > > > > > > > > > Does this code allow you to modify it at all
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > it should since the other .empty() work the same
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Double check since I think this might have been broken for a while
> > > > > > > > 
> > > > > > > > 
> > > > > > > > it works as long as you dont clear out the entire input
> > > > > > > 
> > > > > > > 
> > > > > > > Got it okay
> > > > > > 
> > > > > > 
> > > > > > Quote replies in github are funny
> > > > > 
> > > > > 
> > > > > im enjoying this too much
> > > > 
> > > > 
> > > > Yeah you can leave it like that
> > > 
> > > 
> > > Okay!
> > 
> > Back in my day this is how we had to do it on discord
> 
> it wasnt as funny tho since > dont stack there

Yeah